pkg/nsx/services/common/types.go

```diff
@@ -18,7 +18,7 @@ const (
 	HashLength                      int    = 8
 	MaxTagLength                    int    = 256
 	MaxIdLength                     int    = 255
-	MaxNameLength                   int    = 255
+	MaxNameLength                   int    = 80
```

Comment:
Better to use specific max length for Subnet display name

Reply:
ok